contributing.md

```diff
@@ -0,0 +1,54 @@
+# Contributing to Niceplaces
```

Comment:
Better to use the name of the repository: Nice Places Website

Reply:
Changed to Nice Places Website